src/gpu.rs

```diff
@@ -0,0 +1,62 @@
+use libc::size_t;
+use std::mem::size_of;
+use streamer::SharedPackets;
+
+#[link(name = "cuda_verify_ed25519")]
+extern "C" {
+    fn ed25519_verify_many(
+        packets: *const u8,
```

Comment:
@sakridge this should be an iovec call, so vector of buffers all with the same layout
[sig, payload, up to 256 bytes][usize]